communications/README.md

```diff
@@ -0,0 +1,11 @@
+# Communications
+
+## Async written first
+
+- If you want to discuss an idea (e.g. new or existing product feature, update a marketing page), comment on or create a new story/epic in Clubhouse. @-mention specific individuals and ask them specific questions. See [Product Development](../product-development/README.md).
+- Ensure your [Clubhouse notifications](https://help.clubhouse.io/hc/en-us/sections/201543416-Notifications) are properly configured. Respond to Tilters within a few days with an answer, or at least say you don't have time currently to do so.
+- If you want to discuss a process change, consider the smallest reasonable change and create a GitHub PR in this repo as a proposal. @-mention and assign specific individuals for review.
+- Only when you have exhausted discussions in Clubhouse and/or GitHub, consider scheduling a meeting with a clear agenda to determine next steps.
```

Comment:
lol, I think the phrase "exhausted discussions" is going to open some scar tissue for some people.

I'm worried that this point is going to be interpreted as "all discussions need to end in agreement, and if they don't end in agreement, you keep scheduling meetings until there is agreement". Which I don't think is that you're intending!

I might break this down into two different cases:

- the case where we're just doing initial ideation and brainstorming, but no work has started and no one has taken ownership of the problem yet

- the case where there is a DRI, and they're responsible for the next steps

and how/why each of those cases would evolve into a meeting

Reply:
@nicks : You are right, the purpose of this sentence was not to convey "getting to agreement". I actually wanted to address a simpler more generic case, which is to schedule a meeting when you need to, because it's getting to difficult to talk over async comments. So I think your two cases are actually _too_ specific, and not necessarily what I want to introduce here. What do you think of this?

```suggestion
- If you are having difficulty understanding each other over Clubhouse and/or GitHub, consider scheduling a meeting with a specific agenda to further discuss. But prefer starting with async communications first.
```

Reply:
sgtm